docs/administrators/operational_procedures.rst

```diff
@@ -33,23 +33,31 @@ Context
 Procedure
 ++++++++++
 
-1. Verify in `project.yml` that `install_mode` is set to `release`.
-2. Freeze all modules with
+1. Verify in ``project.yml`` that ``install_mode`` is set to ``release``.
+2. Freeze all modules with:
 
 .. code-block:: bash
 
-    inmanta -vv -X project freeze --recursive --operator "=="
+    inmanta -vv project freeze --recursive --operator "=="
```

Comment:
Have you verified how this behaves for v2 modules? Does it add the constraints to `project.yml`? Are they taken into account by `project install`?

Reply:
Seems to work fine, tested with the `inmanta-module-dummy`